enhancements/monitoring/metrics-collection-profiles.md

```diff
@@ -0,0 +1,437 @@
+---
+title: metrics-collection-profiles
+authors:
+  - @JoaoBraveCoding
+reviewers:
+  - @openshift/openshift-team-monitoring
+approvers:
+  - TBD
+api-approvers: "None"
+creation-date: 2022-12-06
+last-updated: 2023-07-24
+tracking-link:
+  - https://issues.redhat.com/browse/MON-2483
+  - https://issues.redhat.com/browse/MON-3043
+---
```

Comment:
from doc view, https://github.com/openshift/enhancements/blob/7a07c6ff5aca6857c3dd8669d6f58054bc64b348/enhancements/monitoring/metrics-collection-profiles.md
![yaml](https://github.com/juzhao/test/assets/22743651/675f4a36-89fb-420f-8983-89e719a666e4)
should be below
```
authors:
  - "@JoaoBraveCoding"
reviewers:
  - "@openshift/openshift-team-monitoring"
```